File: platform-keyboard/MainActivity.java
package com.yourcompany.yourapp;

import android.view.KeyEvent;
import com.facebook.react.ReactActivity;
import com.expensify.reactnativekeycommand.KeyCommandModule;

public class MainActivity extends ReactActivity {
  @Override
  protected String getMainComponentName() {
    return "main";
  }

  // ⬇️ Forward hardware keys to JS
  @Override
  public boolean onKeyDown(int keyCode, KeyEvent event) {
    KeyCommandModule.getInstance().onKeyDownEvent(keyCode, event);
    return super.onKeyDown(keyCode, event);
  }
}

File: platform-keyboard/MainActivity.kt
package com.yourcompany.yourapp

import android.view.KeyEvent
import com.facebook.react.ReactActivity
import com.expensify.reactnativekeycommand.KeyCommandModule

class MainActivity : ReactActivity() {
  override fun getMainComponentName(): String = "main"

  // ⬇️ Forward hardware keys to JS
  override fun onKeyDown(keyCode: Int, event: KeyEvent?): Boolean {
    KeyCommandModule.getInstance().onKeyDownEvent(keyCode, event)
    return super.onKeyDown(keyCode, event)
  }
}